### PR TITLE
[LLM] Pixtral example

### PR DESCRIPTION
**Pull request by Michaelvll**
Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
  - [x] on GCP
  - [x] on RunPod
  - [x] `sky serve up`
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `conda deactivate; bash -i tests/backward_compatibility_tests.sh`

### CODE REVIEW

File: llm/pixtral/README.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+# Deploy Pixtral with One Click on Any Cloud or Kubernetes
```
Comment (concretevitamin) [inline llm/pixtral/README.md:1]: Deploy Pixtral On Kubernetes or Your Own Cloud

File: llm/pixtral/README.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  ███████████████░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+  ██████████░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░  ░░░░░░░
+      ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░      ░░░
+            ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░    ░░░░░░░░░░░░░░░░░░
+               ░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+                    ░░░░░░░░░░░░░░░░░
+                       ░░░░░
+```
+
+On Sep 11, 2024, Mistral released a new [Pixtral 12B](https://x.com/MistralAI/status/1833758285167722836), their first multimodal model. This guide shows how to use run and deploy this multimodal model on any cloud and Kubernetes.
```
Comment (concretevitamin) [inline llm/pixtral/README.md:48]:
```suggestion
On Sep 11, 2024, Mistral released [Pixtral 12B](https://x.com/MistralAI/status/1833758285167722836), their first multimodal model. This guide shows how to use easily run and deploy this multimodal model on your own cloud or Kubernetes cluster.
```
"their first multimodal model supporting vision and text inputs"?

File: llm/pixtral/README.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+## Run Pixtral on Any Cloud or Kubernetes
+
+1. Install SkyPilot on your local machine:
```
Comment (concretevitamin) [inline llm/pixtral/README.md:52]: link to install page

add `sky check`?

File: llm/pixtral/README.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+```
+
+![Pixtral on SkyPilot](https://i.imgur.com/afzNiaK.png)
+
+
+3. Get the endpoint and send requests:
+```bash
+ENDPOINT=$(sky status --endpoint 8081 pixtral)
+
+curl --location http://$ENDPOINT/v1/chat/completions \
```
Comment (concretevitamin) [inline llm/pixtral/README.md:70]: we don't need --location now?

File: llm/pixtral/README.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+```
+
+On Sep 11, 2024, Mistral released a new [Pixtral 12B](https://x.com/MistralAI/status/1833758285167722836), their first multimodal model. This guide shows how to use run and deploy this multimodal model on any cloud and Kubernetes.
+
+## Run Pixtral on Any Cloud or Kubernetes
+
+1. Install SkyPilot on your local machine:
+
+```bash
+pip install 'skypilot-nightly[all]'
```
Comment (concretevitamin) [inline llm/pixtral/README.md:55]: Is stable ok? Or must use nightly?

File: llm/pixtral/README.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+  "prompt_logprobs": null
+}
+```
+
+</details>
+
+
+## Scale Up Pixtral Endpoint as a Service
+
+1. Start a service with SkyServe:
```
Comment (concretevitamin) [inline llm/pixtral/README.md:147]: Link to docs

File: llm/pixtral/README.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+```bash
+sky serve up -n pixtral pixtral.yaml
+```
+
+2. Get the endpoint and send requests:
+```bash
+ENDPOINT=$(sky serve status --endpoint pixtral)
+
+curl --location http://$ENDPOINT/v1/chat/completions \
```
Comment (concretevitamin) [inline llm/pixtral/README.md:156]: need -L?

File: llm/pixtral/pixtral.yaml [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  readiness_probe:
+    path: /v1/chat/completions
+    post_data:
+      model: $MODEL_NAME
+      messages:
+        - role: user
+          content: {
+            "type": "text",
+            "text": "What's in this image?"
+          }
```
Comment (concretevitamin) [inline llm/pixtral/pixtral.yaml:16]: A bit confusing. No image in prompt.

File: docs/source/_gallery_original/index.rst [hunk 9]
```diff
@@ -34,6 +34,7 @@ Contents
    :maxdepth: 1
    :caption: LLM Models
 
+   Pixtral (Mistral AI) <llms/pixtral>
```
Comment (romilbhardwaj) [inline docs/source/_gallery_original/index.rst:37]: Should we put `new` label in custom.js?

File: llm/pixtral/README.md (hunk 2 above)
Comment (romilbhardwaj) [reply llm/pixtral/README.md:48]:
```suggestion
On Sep 11, 2024, Mistral released [Pixtral 12B](https://x.com/MistralAI/status/1833758285167722836), their first multimodal model. This guide shows how to easily run and deploy this multimodal model on your own cloud or Kubernetes cluster.
```

File: llm/pixtral/README.md [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+            "content": [
+                {"type" : "text", "text": "Describe this image in detail please."},
+                {"type": "image_url", "image_url": {"url": "https://s3.amazonaws.com/cms.ipressroom.com/338/files/201808/5b894ee1a138352221103195_A680%7Ejogging-edit/A680%7Ejogging-edit_hero.jpg"}},
+                {"type" : "text", "text": "and this one as well."},
+                {"type": "image_url", "image_url": {"url": "https://www.wolframcloud.com/obj/resourcesystem/images/a0e/a0ee3983-46c6-4c92-b85d-059044639928/6af8cfb971db031b.png"}}
+            ]
+        }],
+        "max_tokens": 1024
+    }' | jq .
+```
```
Comment (romilbhardwaj) [inline llm/pixtral/README.md:173]: Should we add response in a dropdown here?

**Review by romilbhardwaj — APPROVED** [review]

**Pull request merged**